go-controller/pkg/factory/factory.go

```diff
@@ -479,6 +483,97 @@ func NewWatchFactory(c kubernetes.Interface, stopChan chan struct{}) (*WatchFact
 	return wf, nil
 }
 
+// NewNodeWatchFactory creates only the informers we need for node-level operations,
+// saving bandwidth and memory
+// It only configures informers for:
+// - pods, and only the pods on this node
+// - services
+// - endpoints
+// - nodes
+func NewNodeWatchFactory(nodeName string, c kubernetes.Interface, stopChan chan struct{}) (*WatchFactory, error) {
+	// resync time is 12 hours, none of the resources being watched in ovn-kubernetes have
```

Comment:
move this comment up to the definition of `resyncInterval` rather than duplicating it?

Reply:
derp. yes.